changelog.d/15318.feature

```diff
@@ -0,0 +1 @@
+Experimental support for MSC3970: Scope transaction IDs to devices.
```

Comment:
Since MSC3970 landed before this, should we just stabilize it so we never have to deal with a released version of the unstable version?